scripts/prep/aqm/exevs_aqm_prep.sh

```diff
@@ -93,15 +108,29 @@ done
 ##
 checkfile=${DCOMINairnow}/${INITDATE}/airnow/daily_data_v2.dat
 if [ -s ${checkfile} ]; then
-    if [ -s ${conf_dir}/Ascii2Nc_daily_obsAIRNOW.conf ]; then
-        run_metplus.py ${conf_dir}/Ascii2Nc_daily_obsAIRNOW.conf ${PARMevs}/metplus_config/machine.conf
-        export err=$?; err_chk
-	if [ ${SENDCOM} = "YES" ]; then
-            cpfile=${PREP_SAVE_DIR}/airnow_daily_${INITDATE}.nc
-            if [ -s ${cpfile} ]; then cp -v ${cpfile} ${COMOUTproc};fi
-	fi
+    screen_file=${DATA}/checked_daily_data_v2.dat
+    python ${USHevs}/${COMPONENT}/screen_airnow_obs_daily.py ${checkfile} ${screen_file}
+    number_of_record=$(wc -l ${screen_file} | awk -F" " '{print $1}')
+    if [ ${number_of_record} -gt 0 ]; then
+        if [ -s ${conf_dir}/Ascii2Nc_daily_obsAIRNOW.conf ]; then
+            run_metplus.py ${conf_dir}/Ascii2Nc_daily_obsAIRNOW.conf ${PARMevs}/metplus_config/machine.conf
```

Comment:
I believe there should be a check here for `$screen_file` before running METplus.